libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1159,9 +1159,7 @@ void TypeChecker::endVisit(TryStatement const& _tryStatement)
 					m_errorReporter.typeError(
 						9908_error,
 						clause.location(),
-						"This catch clause type cannot be used on the selected EVM version (" +
-						m_evmVersion.name() +
-						"). You need at least a Byzantium-compatible EVM or use `catch { ... }`."
+						"This catch clause type cannot be used on the selected EVM version. You need at least a Byzantium-compatible EVM or use `catch { ... }`."
```

Comment:
Why this change? And the one below? Just for collapsing multiple test cases in the end? If it's that we could also split the tests instead...

Reply:
If there's a tradeoff to be made between a more explicit error message and more uniform testing, I'd say it's in favour of a more explicit error message (even if that means duplicating test cases for each evm version we want to test). But it's not too important, so fine either way.

Reply:
Agree. I will split the test cases.